Change the default font to one very likely to be present on a Debian box

Index: vlc-debian/modules/misc/freetype.c
===================================================================
--- vlc-debian.orig/modules/misc/freetype.c	2009-07-09 00:21:18.000000000 +0200
+++ vlc-debian/modules/misc/freetype.c	2009-07-09 00:23:16.000000000 +0200
@@ -60,7 +60,7 @@
 #define DEFAULT_FONT "" /* Default font found at run-time */
 #define FC_DEFAULT_FONT "Arial"
 #else
-#define DEFAULT_FONT "/usr/share/fonts/truetype/freefont/FreeSerifBold.ttf"
+#define DEFAULT_FONT "/usr/share/fonts/truetype/ttf-dejavu/DejaVuSans-Bold.ttf"
 #define FC_DEFAULT_FONT "Serif Bold"
 #endif
 
